crates/storage-api/proto/dev/restate/storage/v1/domain.proto

```diff
@@ -121,10 +115,7 @@ message InvocationStatus {
         uint64 creation_time = 4;
         uint64 modification_time = 5;
 
-        oneof idempotency_key {
-            google.protobuf.Empty idempotency_key_none = 12;
-            string idempotency_key_value = 13;
-        }
+        optional string idempotency_key = 12;
```

Comment:
Here we are gonna have some nice merge conflicts with one of my PRs 😅

Reply:
:fearful: